docs/BasicCommunication/GetSystemTime/index.md

```diff
@@ -0,0 +1,109 @@
+## GeSystemTime
+
+Type
+: Function
+
+Sender
+: SDL
+
+Purpose
+: Obtain current UTC time  
+
+When an application  wants to open protected service it sends certificate to SDL during TLS or DTLS handshake.  
+In order to validate the certificate and to check whether this certificate is not expired SDL needs system time.
+
+### Request
+SDL sends GetSystemTime request after receiving BC.OnSystemTimeReady from HMI.  
+After sending the request SDL starts [`<DefaultTimeout>`](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/develop/docs/Configuration%20file/index.md#main) (value from .ini file) waiting for response from HMI. 
+
+#### Parameters
+
+### Response
+
+!!! must  
+
+1.	Send valid response during `<DefaultTimeout>`(value from .ini file)
```

Comment:
1.	Send valid response during `<DefaultTimeout>`(value from .ini file)
to
1.	Send a valid response during the `<DefaultTimeout>`(value from the smartDeviceLink.ini file)

Reply:
@mrapitis, thank you.
fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/94/commits/3c7a300842699307e9fd06ffb64fcd65d66d6d77